portfolio-project-matching-app/components/ProjectCard.js

```diff
@@ -17,32 +17,36 @@ const ProjectCard = ({ project }) => {
                 <p className='text-xl font-medium'>{project.name}</p>
             </div>
             <hr className='w-11/12 sm:w-9/12 border-b-2 border-gray-400'/>
-            <div className='flex flex-wrap'>
+            {/* Commented out, but leaving in just in case we decide to add this back */}
+            {/* <div className='flex flex-wrap'>
                 {project.technologies.map((technology) => {
                     return (
                         <div key={technology.id} className='py-1 pr-2'>
                             <Button text={technology.name}/>
                         </div>
                 )
                 })}
-            </div>
+            </div> */}
```

Comment:
Did away with the technology listing.

Reply:
So if we remove the technologies from the project card, it seems like we also are getting rid of the ability to filter based on technology? This seems like a relatively big feature that we would be removing from the `browseProjects` page.